include/aspect/material_model/interface.h

```diff
@@ -180,6 +182,16 @@ namespace aspect
       MaterialModelInputs(const unsigned int n_points,
                           const unsigned int n_comp);
 
+
+      //**************************************
```

Comment:
This comment block should be of the same style as the other comment blocks (especially starting with `/**`). It is with this formatting that the documentation generation is able to grab your description.